tests/test-rgw.yml

```diff
@@ -0,0 +1,15 @@
+---
+- hosts: localhost
```

Comment:
Lets change this to run against only the intersection of hosts (which will target osdmonrgw):

hosts: osds:&mons:&rgws

Alternatively if this will work on any mon host we could do:
hosts: mons[0]

I think that way we can avoid having to run the "lxc-attach -n" command each time.